src/colorize.ts

```diff
@@ -22,49 +18,55 @@ export default (
 		return str;
 	}
 
-	if (color in chalk) {
-		const method = getMethod(color, type);
-		return (chalk as any)[method](str);
+	if (isNamedColor(color)) {
+		if (type === 'foreground') {
+			return chalk[color](str);
+		}
+
+		const methodName = `bg${
+			color[0]!.toUpperCase() + color.slice(1)
```

Comment:
```suggestion
			color[0]!.toUpperCase() + color.slice(1)
```

I believe our maintainers don't want `!.` anywhere.